libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationView.kt

```diff
@@ -447,6 +491,22 @@ class NavigationView : ConstraintLayout {
         }
     }
 
+    // This was added to facilitate getting a route into mapbox navigation so work could go forward.
+    // It may be temporary.
+    internal val onMapLongClickListener = OnMapLongClickListener { point ->
+        val currentLocation = navigationLocationProvider.lastLocation
+        if (currentLocation != null) {
+            val originPoint = Point.fromLngLat(
+                currentLocation.longitude,
+                currentLocation.latitude
+            )
+            mapboxNavigationViewModel.findRoute(originPoint, point, this@NavigationView.context)
```

Comment:
We cannot pass in the view context to the `viewmodel` as it would leak on screen orientation. It's probably better to pass in the `RouteOptions`.
/cc @cafesilencio

Reply:
This is not a blocker but can be improved.

Reply:
This is just temporary to get a route into NavigationView. The reference to the context isn't held but it's needed to fetch a route.